tests/core/utilities.py

```diff
@@ -85,3 +86,7 @@ def user_uttered(
 
 def get_tracker(events: List[Event]) -> DialogueStateTracker:
     return DialogueStateTracker.from_events("sender", events, [], 20)
+
+
+class CustomRedisLockStore(RedisLockStore):
+    pass
```

Comment:
this class should not be here - it's only needed for tests

Reply:
I placed this class inside the test file. Let me know if that works!